deploy-0chain/action.yml

```diff
@@ -337,6 +337,7 @@ runs:
              --set commonConf.sc.smart_contracts.faucetsc.periodic_limit=10000000 \
              --set commonConf.sc.smart_contracts.faucetsc.global_limit=100000000 \
              --set commonConf.sc.smart_contracts.faucetsc.max_pour_amount=10000 \
+             --set commonConf.sc.smart_contracts.storagesc.max_challenge_completion_rounds=1000 \
```

Comment:
Hey Jayash, could you please explain why this needs to be separate from prod value for testing, thanks

Reply:
@Jayashsatolia403 please don't resolve with replying

Reply:
It is new config item. Now we will expire challenges on basis of rounds. 

Really sorry for inconvenience.